src/main/java/GoblintResult.java

```diff
@@ -12,10 +12,16 @@
 
 public class GoblintResult {
 
+    private List<tag> tags = new ArrayList<>();
     private String severity;
     private multipiece multipiece;
     URL sourcefileURL;
 
+    static class tag {
+        private List<String> Category = new ArrayList<String>();
+        private Integer CWE;
+    }
```

Comment:
This structure is a bit weird because it looks like a single tag may be both a category tag and a CWE tag, but that's actually not the case from Goblint. To be precise, I guess tag should be an interface with two subclasses: category and CWE. Then their `toString` could implement the `findTags()` logic independently.

Although it's probably more difficult to get Gson to deserialize into an interface based on the content that would go to either subclass, so for the time being this is good enough to use.